app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -281,13 +282,54 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
             actions: ["transcribe:StartStreamTranscriptionWebSocket","transcribe:StartMedicalStreamTranscription","comprehendmedical:InferICD10CM","comprehendmedical:InferRxNorm","comprehendmedical:DetectEntitiesV2"]
           })
         );
+        
+        // Dynamodb
+        const TableSessions = new ddb.Table(this, 'Table1', {
+          tableName: 'Sessions',
+          partitionKey: { name: 'PatientId', type: ddb.AttributeType.STRING },
+          sortKey: {name: 'SessionId', type: ddb.AttributeType.NUMBER },
+          serverSideEncryption: true
+        });
+
+        TableSessions.addGlobalSecondaryIndex({
+          indexName: "hcpIndex",
+          partitionKey: { name: 'HealthCareProId', type: ddb.AttributeType.STRING },
+          sortKey: {name: 'SessionId', type: ddb.AttributeType.NUMBER }
+        });
+        
+        const TablePatients = new ddb.Table(this, 'Table2', {
+          tableName: 'Patients',
+          partitionKey: { name: 'PatientId', type: ddb.AttributeType.STRING },
+          serverSideEncryption: true
+        });
+        
+        const TableHealthCareProfessionals = new ddb.Table(this, 'Table3', {
+          tableName: 'HealthCareProfessionals',
+          partitionKey: { name: 'HealthCareProId', type: ddb.AttributeType.STRING },
+          serverSideEncryption: true
+        });
 
         // Lambda
+        const createHealthCareProAPI = new lambda.Function(
+          this,
+          this.resourceName("CreateHealthCareProAPI"),
+          {
+            runtime: lambda.Runtime.PYTHON_3_8,
+            code: lambda.Code.asset("lambda/create_health_care_professional"),
+            handler: "lambda_function.lambda_handler",
+            timeout: cdk.Duration.seconds(60),
+            environment: {
+              TRANSCRIBE_ACCESS_ROLEARN: transcriberRole.roleArn,
```

Comment:
Transcribe Role Arn, Region were passed as an environment variable in the Get Credentials Lambda since we needed to assume that role. I dont think we need this environment variable in this Lambda. Could you remove it?